extensions/amp-ad-network-doubleclick-impl/0.1/amp-ad-network-doubleclick-impl.js

```diff
@@ -486,6 +508,11 @@ export class AmpAdNetworkDoubleclickImpl extends AmpA4A {
       'frc': Number(this.fromResumeCallback) || null,
       'fluid': this.isFluidRequest_ ? 'height' : null,
       'fsf': this.forceSafeframe ? '1' : null,
+      'msz': this.sendFlexibleAdSlotParams_ ?
+        `${getContainerWidth(this.win, this.element)}x-1` : null,
+      'psz': this.sendFlexibleAdSlotParams_ ?
+        `${getContainerWidth(this.win, this.element.parentElement)}x-1` :
+        null,
```

Comment:
comment here that we're intentionally sending `${width}x-1` because height can't change?

Reply:
Done.